src/Groups/action.jl

```diff
@@ -669,6 +669,41 @@ function stabilizer(G::MatrixGroup{ET,<:MT}, pnt::MatElem{<:MT}, actfun::Functio
     GAP.Globals.OnSubspacesByCanonicalBasis))
 end
 
+# action of a group G on class functions of a group H that is normalized by G,
+# defined by $\chi^g: h \mapsto \chi(conj(h, inv(g)))$
+function stabilizer(G::GAPGroup, chi::GAPGroupClassFunction)
+  tbl = parent(chi)
+  actions = get_attribute!(IdDict{GAPGroup,GAPGroupHomomorphism}, tbl,
+                           :actions_on_classes)
+  if haskey(actions, G)
+    phi = actions[G]
+    img = codomain(phi)
+  else
+    ccl = conjugacy_classes(tbl)
+    n = length(ccl)
+    sym = symmetric_group(n)
+    imgs = PermGroupElem[]
+    reps = map(representative, ccl)
+    Ggens = gens(G)
+    for g in Ggens
+      l = Int[]
+      for r in reps
+        y = r^g
+        pos = findfirst(c -> y in c, ccl)
+        @req pos !== nothing "G does not normalize the group of chi"
```

Comment:
Discussion point: if this helpful, or should this be "1st argument does not normalize the group of the 2nd"?
I am always confused if an error message mentions variables that I do not have.
But this is for discussion in triage. I am happy with this pr

Reply:
We can give a hint in our General/'Frequently Asked Questions' about consulting Julia's stacktrace in case of errors.

And for GAP users (who are used to inspect variables inside the break loop in case of errors), we can give such a hint in General/'Notes for users of other computer algebra systems'.